nmostesting/suites/IS1101Test.py

```diff
@@ -1715,3 +1725,571 @@ def test_04_02_02(self, test):
+                return test.FAIL("Unexpected response from the streamcompatibility API: {}".format(response))
+            if response.status_code != 200:
+                return test.FAIL("The streamcompatibility request for output {} properties has failed: {}"
+                                 .format(output_id, response.json()))
+            if not (CONFIG.IS11_REFERENCE_SENDER_CONNECTION_API):
+                return test.DISABLED("Please configure IS11_REFERENCE_SENDER_CONNECTION_API in Config.py")
+            if (response.json()["status"]["state"] != "signal_present"):
+                return test.FAIL("The state: {} should be signal_present".format(response.json()["status"]["state"]))
+            if len(self.senders) == 0:
+                return test.DISABLED("No IS-11 reference sender")
+            for sender in self.senders:
+                new_netloc = "{}:{}".format(CONFIG.IS11_REFERENCE_SENDER_CONNECTION_API,
+                                            CONFIG.IS11_REFERENCE_SENDER_CONNECTION_API_PORT)
+                url = self.conn_url.replace(urlparse(self.conn_url).netloc, new_netloc)
+                json_data = {
+                    "master_enable": False,
+                    "activation": {"mode": "activate_immediate"}
+                }
+                headers = {"Content-Type": "application/json"}
+                valid_patch, response = TestHelper.do_request('PATCH', url + "single/senders/"
+                                                              + sender["id"] + "/staged/",
+                                                              json=json_data, headers=headers)
+                if not valid_patch:
+                    return test.FAIL("Unexpected response from the Connection API: {}".format(response))
+                if response.status_code != 200:
+                    return test.FAIL("The patch request to {} has failed: {}"
+                                     .format(url + "single/senders/" + sender["id"] + "/staged/", response.json()))
+                time.sleep(CONFIG.API_PROCESSING_TIMEOUT)
+        return test.PASS()
+
+    def test_04_04(self, test):
+        """
+        Verify receivers not supporting outputs
+        """
+        if len(self.receivers) == 0:
+            return test.DISABLED("No IS-11 receivers")
+        for receiver_id in self.receivers:
+            _, response = TestHelper.do_request('GET', self.compat_url + "receivers/" + receiver_id + "outputs/")
+            if response.status_code != 200:
+                return test.FAIL("The streamcompatibility request for receiver {} outputs has failed: {}"
+                                 .format(receiver_id, response.json()))
+            if len(response.json()) == 0:
+                self.no_output_receivers.append(receiver_id)
+            if len(self.no_output_receivers) == 0:
+                return test.FAIL("All IS-11 receivers support outputs")
+        return test.PASS()
+
+    def test_04_04_01(self, test):
```

Comment:
It also requires a check of media type, right?

Reply:
This was reworked. Let me know if this works for you now.